cds-snc/cppd-medical-report
# Supporting documents

# Description

Add Supporting Documents route.

Depends on #71 - merge that one first

# Test Instructions

- With application running locally, visit /en/documents
- Submit the form without filling it in to observe the validation
- Submit the completed form and it will take you to the confirmation screen
